### ISSUE TITLE
How to set the parameter of lr_step

### ISSUE THREAD

**Issue by kinfeparty** [okankop/Efficient-3DCNNs#31]
Hello!  I want to train the model in Jester dataset.  I found that the lr_step in your code is  [45, 60, 50, 200, 250].

But the code in   https://github.com/ahmetgunduz/Real-time-GesRec/blob/master/opts.py    is      [15, 25, 35, 45, 60, 50, 200, 250].

Do you train the model with different lr_step?   If I want to train my model such as large model (resnext101、resnet50) ,Which lr_step should I use?

**Comment by okankop** [comment]
For Jester dataset, you can use the lr steps of  [15, 25, 35] and stop training after 40 epochs. You will observe that at these lr_steps, the validation loss already converges.
